index.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 class CancelError extends Error {
-	constructor() {
-		super('Promise was canceled');
+	constructor(reason = '') {
+		super('Promise was canceled. Reason: ' + reason);
```

Comment:
Maybe the provided reason should just replace the message entirely? It's already clear from the error name that it was canceled.